### PR TITLE
Some more documentation on the headless mode.

### PR DESCRIPTION
**Pull request by hno2**
- Add ReadOnly=1 option
- Add Link to the python client example included in ilastik.

### CODE REVIEW

File: documentation/basics/headless.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ the command (as shown in the example above).
 - `--output_axis_order` Transpose the storage order of the results. For example, this affects the sliced dimension for stack outputs.
 - `--pipeline_result_drange` Pipeline result data range (min,max) BEFORE normalization, e.g. `"(0.0,1.0)"`
 - `--export_drange` Exported data range (min,max) AFTER normalization, e.g. `"(0,255)"`
+- `--readOnly=1` Open project in read-only mode to allow multiple processing accessing the same file.
```
Comment (emilmelnikov) [inline documentation/basics/headless.md]:
```suggestion
- `--readonly` Open the project in read-only mode; necessary for using a single project from multiple processes.
```

File: documentation/basics/headless.md [hunk 2]
```diff
@@ -228,7 +229,7 @@ For developers and power-users, you can run your own ilastik-dependent python sc
     # Mac
     $ ./ilastik-1.3.2-OSX.app/Contents/ilastik-release/bin/python -c "import ilastik; print ilastik.__version__"
     1.3.2
-
+Check out [this example](https://github.com/ilastik/ilastik/blob/master/examples/example_python_client.py) on how to use ilastik as a python module
```
Comment (k-dominik) [inline documentation/basics/headless.md]: We plan on changing the way to use ilastik as a python module completely and don't want people to start building projects now, relying on the old way.
```suggestion
```

**Comment by k-dominik** [comment]
Hi @hno2,

thanks a lot for adding the note. As mentioned in the comment, we would prefer not to point to the python client example anymore.

File: documentation/basics/headless.md (hunk 2 above)
Comment (hno2) [reply documentation/basics/headless.md]: Oh I'm sorry I didn't know that. I am currently building a project relying on that and it took some time for me to find this example.

**Pull request merged**